src/main/java/seedu/address/model/patient/Record.java

```diff
@@ -51,11 +59,7 @@ public void setTreatmentPlan(String treatmentPlan) {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this)
-                .add("patient", patient)
-                .add("initialObservations", initialObservations)
-                .add("diagnosis", diagnosis)
-                .add("treatmentPlan", treatmentPlan)
-                .toString();
+        return new ToStringBuilder(this).add("patient", patient).add("initialObservations", initialObservations)
+            .add("diagnosis", diagnosis).add("treatmentPlan", treatmentPlan).toString();
```

Comment:
Not sure about the indentation here but I feel like the one before looks a bit neater? 😅

Reply:
Would fix the indentation one shot after further changes, thanks for pointing out!